### PR TITLE
add ability to statically retrieve a data models rules

### PR DESCRIPTION
**Pull request by abenerd**
I sometimes only need the data models rules before working witht the model itself, this allows me to easily access those rules

### CODE REVIEW

File: src/Concerns/ValidateableData.php [hunk 1]
```diff
@@ -52,4 +52,17 @@ public static function withValidator(Validator $validator): void
     {
         return;
     }
+
```
Comment (rubenvanassche) [inline src/Concerns/ValidateableData.php:55]: Could you rename this to `getValidationRules`?

**Review by rubenvanassche — CHANGES_REQUESTED** [review]
Small comment, could you also please add a small section to the docs, describing this feature? Might be handy for other people in the future!

**Comment by abenerd** [comment]
Sure, I will do so tomorrow!

File: src/Concerns/ValidateableData.php (hunk 1 above)
Comment (abenerd) [reply src/Concerns/ValidateableData.php:55]: Yeah, will also do that tomorrow with the documentation.

**Comment by abenerd** [comment]
I added some documentation and updated the method name @rubenvanassche.

**Comment by rubenvanassche** [comment]
Thanks!

**Pull request merged**